> Yeah I don't understand why that test is failing and as I'm fairly busy with other things maybe someone more familiar with the domain should take over.

I'm just gonna hit the rebuild button, because there's approximately a zero percentage change of it being related to this PR 😇

for the record:

```
[2450/2450] webkitpy.port.server_process_unittest.TestServerProcess.serial_test_process_crashing failed:
  Traceback (most recent call last):
    File "/Volumes/Data/worker/WebKitPy-Tests-EWS/build/Tools/Scripts/webkitpy/port/server_process_unittest.py", line 179, in serial_test_process_crashing
      self.assertEqual(True, proc.has_crashed())
  AssertionError: True != False
  ```

--- a/Tools/Scripts/webkitpy/w3c/test_downloader.py
+++ b/Tools/Scripts/webkitpy/w3c/test_downloader.py
@@ -144,7 +144,7 @@ class TestDownloader(object):
             if not already_imported:
                 import_lines[stripped_path] = "import"
 
-        self._filesystem.write_text_file(self.import_expectations_path, json.dumps(import_lines, sort_keys=True, indent=4, separators=(',', ': ')))
+        self._filesystem.write_text_file(self.import_expectations_path, json.dumps(import_lines, sort_keys=True, indent=4, separators=(',', ': ')) + "\n")
 
     def clone_tests(self, use_tip_of_tree=False):
         for test_repository in self.test_repositories:

